.github/actions/goreleaser-build-sign-publish/README.md

```diff
@@ -0,0 +1,125 @@
+# goreleaser-build-sign-publish
+> goreleaser wrapper action
+
+## workflows
+
+### build publish
+
+```yaml
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    environment: release
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
Should pin this

Reply:
for examples in the readme, I dont think we should pin, I checked examples from other action docs